### PR TITLE
YG-311: Manage Classroom and Day access as per the user type

### PR DESCRIPTION
**Pull request by pepcus-akshita**
# Related issues
- Manage Classroom and Day access as per the user type

# Screenshots
Scenario 1 - 

https://user-images.githubusercontent.com/113973515/231448893-ca6a43bc-3719-46ac-99b9-f83a5dec3c5e.mp4

Scenario 2 - 

https://user-images.githubusercontent.com/113973515/231448406-c4ececeb-f0ec-4d85-b8c5-2ec0da7d30b1.mp4

### CODE REVIEW

File: src/components/routeComponents/markAttendance/FetchRoomAttendanceForm.jsx [hunk 1]
```diff
@@ -1,29 +1,82 @@
...
+	const fetchClassRoomList = async () => {
+		setLoader(true);
+		const apiRequest = {
+			...apiConfigFetchClassRoom,
+			additionalData: { secretKey: AUTHTOKEN.secretKey },
+		};
+		return await callAPIWithConfig(tenant, 'fetchClassRoom', apiRequest);
+	};
+
+	console.log('# formdata in fetchroomattendance', formData);
```
Comment (pepcus-mohit) [inline src/components/routeComponents/markAttendance/FetchRoomAttendanceForm.jsx:75]: Remove console
```suggestion
	
```

File: ui_config/tenant/YJSG-Indore/routes/splashPage.json [hunk 2]
```diff
@@ -127,18 +121,9 @@
...
-						"4",
-						"5",
-						"6",
-						"7",
-						"8"
-					],
+					"enum": ["1", "2", "3", "4", "5", "6", "7", "8"],
 					"enumNames": [
-						"Day 1 (30-04-2023)",
+						"Day 1 (07-04-2023)",
```
Comment (pepcus-akshita) [inline ui_config/tenant/YJSG-Indore/routes/splashPage.json:126]: Changed the date for testing purpose only for one day.
Change Require before merging.

File: ui_config/tenant/YJSG-Indore/routes/splashPage.json [hunk 3]
```diff
@@ -227,6 +204,106 @@
...
+				"membersId": {
+					"title": "Add other student Ids (optional)",
+					"type": "string"
+				},
+				"selectedDay": {
+					"title": "Day:",
+					"type": "string",
+					"enum": ["1", "2", "3", "4", "5", "6", "7", "8"],
+					"enumNames": [
+						"Day 1 (07-04-2023)",
```
Comment (pepcus-akshita) [inline ui_config/tenant/YJSG-Indore/routes/splashPage.json:226]: Changed the date for testing purpose only for one day.
Change Require before merging.

File: ui_config/tenant/YJSG-Indore/routes/splashPage.json (hunk 2 above)
Comment (pepcus-mohit) [inline ui_config/tenant/YJSG-Indore/routes/splashPage.json:126]: This should not be changed.
Also Day 8 should be "Day 8 (08-05-2023)"

File: ui_config/tenant/YJSG-Indore/routes/splashPage.json
Comment (pepcus-mohit) [inline ui_config/tenant/YJSG-Indore/routes/splashPage.json:1]: JSON formatting?

File: ui_config/tenant/YJSG-Indore/routes/splashPage.json [hunk 4]
```diff
@@ -227,6 +204,106 @@
...
+						"Day 2 (01-05-2023)",
```
Comment (pepcus-mohit) [inline ui_config/tenant/YJSG-Indore/routes/splashPage.json:227]:
```suggestion
						"Day 2 (30-04-2023)",
```

File: ui_config/tenant/YJSG-Indore/routes/splashPage.json [hunk 5]
```diff
@@ -227,6 +204,106 @@
...
+						"Day 3 (02-05-2023)",
+						"Day 4 (03-05-2023)",
+						"Day 5 (04-05-2023)",
+						"Day 6 (05-05-2023)",
+						"Day 7 (06-05-2023)",
+						"Day 8 (07-05-2023)"
```
Comment (pepcus-mohit) [inline ui_config/tenant/YJSG-Indore/routes/splashPage.json:233]:
```suggestion
						"Day 8 (08-05-2023)"
```

**Review by pepcus-mohit — CHANGES_REQUESTED** [review]

**Review by pepcus-mohit — APPROVED** [review]

**Pull request merged**